runtime/service.go

```diff
@@ -666,6 +699,22 @@ func (s *service) stopVM() error {
 	return s.machine.StopVMM()
 }
 
+func parseCreateTaskOpts(ctx context.Context, opts *ptypes.Any) (*proto.FirecrackerConfig, error) {
+	log.G(ctx).WithField("opts", opts).Debug("unmarshaling create task create req options")
+
+	cfg, err := typeurl.UnmarshalAny(opts)
+	if err != nil {
+		return nil, errors.Wrapf(err, "unmarshaling task create request options")
+	}
+
+	firecrackerConfig, ok := cfg.(*proto.FirecrackerConfig)
```

Comment:
I think we should also expect runc options here.  I'd like to avoid requiring client libraries to use the `proto.FirecrackerConfig` type if they want to use standard runc options and don't have anything Firecracker-specific to add.

Reply:
Hmm, so essentially handle  two types here? I can work with that. I'll modify to handle that scenario.